src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -3566,23 +3622,18 @@ void InitIntSolarDistribution(EnergyPlusData &state)
         for (int SurfNum = firstSurfOpaque; SurfNum <= lastSurfOpaque; ++SurfNum) {
             int const ConstrNum = Surface(SurfNum).Construction;
 
-            Real64 AbsIntSurf = state.dataConstruction->Construct(ConstrNum).InsideAbsorpSolar; // Inside opaque surface solar absorptance
+            Real64 AbsIntSurf = state.dataHeatBalSurf->SurfAbsSolarInt(SurfNum);
+            // TODO - AbsIntSurfVis = InsideAbsorpSolar instead of InsideAbsorpVis?
```

Comment:
If I'm following correctly, `AbsIntSurfVis` is only used for reporting, not in the actual surface heat gain calculations. Per the comments in [CR8695](https://github.com/NREL/EnergyPlus/issues/3468) it sounds like these needs to remain AbsSolar so that the reporting is consistent with the actual heat balance. But it's certainly worth looking at. It might be possible to fix that (later).